src/Akeneo/Pim/Enrichment/Bundle/Controller/Ui/CategoryTreeController.php

```diff
@@ -426,6 +432,11 @@ protected function buildAclName($name)
         return $this->rawConfiguration['acl'] . '_' . $name;
     }
 
+    public function getTemplateUuid(CategoryInterface $OrmCategory): ?TemplateUuid
```

Comment:
fixing my own mistakes ^^'
```suggestion
    public function getTemplateUuid(CategoryInterface $ormCategory): ?TemplateUuid
```